### PR TITLE
Case list eventing

### PR DESCRIPTION
**Pull request by sshumake**
https://projects.engineering.redhat.com/browse/PCM-438

### CODE REVIEW

**Comment by sshumake** [comment]
@renujhamtani @pbathia  please review/merge

File: app/cases/controllers/group.js [hunk 1]
```diff
@@ -20,5 +29,9 @@ angular.module('RedhatAccess.cases').controller('Group', [
         $scope.defaultCaseGroup = function(){
             $location.path('/case/group/default');
         };
+        $scope.$on('$destroy', function () {
```
Comment (pbathia) [inline app/cases/controllers/group.js]: @sshumake Instead of defining new destroy function, you can use existing destroy function at line 26.

**Comment by pbathia** [comment]
@sshumake Rest all code looks fine to me, except changes mentioned above in group.js.

**Comment by renujhamtani** [comment]
@sshumake @pbathia Agreed with Pranjal's comments. Once the review comments are incorporated, I will merge this.

**Pull request merged**